src/SignalR/common/Http.Connections/src/Internal/HttpConnectionManager.cs

```diff
@@ -65,14 +65,14 @@ internal bool TryGetConnection(string id, [NotNullWhen(true)] out HttpConnection
 
         internal HttpConnectionContext CreateConnection()
         {
-            return CreateConnection(PipeOptions.Default, PipeOptions.Default);
+            return CreateConnection(new HttpConnectionDispatcherOptions());
```

Comment:
Does this work:

```
            return CreateConnection(new());
```

Reply:
https://twitter.com/davidfowl/status/1389128645482926080?s=20